code/src/WijmoProvider/Columns/CalculatedColumn.ts

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace WijmoProvider.Column {
+    export class CalculatedColumn extends AbstractProviderColumn<OSFramework.Configuration.Column.ColumnConfigAdditional> {
+        constructor(
+            grid: OSFramework.Grid.IGrid,
+            columnID: string,
+            configs: JSON,
+            extraConfig: JSON
+        ) {
+            super(
+                grid,
+                columnID,
+                new OSFramework.Configuration.Column.ColumnConfigAdditional(
+                    configs,
+                    extraConfig
+                )
+            );
+            this._columnEvents =
+                new OSFramework.Event.Column.ColumnEventsManager(this);
+
+            // set custom binding
```

Comment:
Could you please just comment here what is the logic to create the binding name?

Reply:
done